client/jetpack-cloud/components/guided-tour/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { Popover, Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { getJetpackDashboardPreference as getPreference } from 'calypso/state/jetpack-agency-dashboard/selectors';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { preferencesLastFetchedTimestamp } from 'calypso/state/preferences/selectors';
+
+import './style.scss';
+
+export interface Tour {
+	id: string;
```

Comment:
Not a blocker, but can we change this to `targetElement` or another name that emphasizes this is mapped to a component?